assets/css/v2/pre_fare/free_text.scss

```diff
@@ -59,3 +59,61 @@
     line-height: unset;
   }
 }
+
+.departures__notice-row {
+  // Some of the same properties as, just copied specifically for departures.
```

Comment:
Maybe worth pulling into a mixin for some of these properties? 

Or maybe there's a better way of handling this specific issue with free-text overlap here and above?

Reply:
I think a `mixin` makes sense here! Could even just be defined earlier in this .scss file